release.sh

```diff
@@ -14,7 +14,7 @@ TAG=$(git describe --exact-match --tags HEAD)
 
 VERSION=${TAG/v/}
 
-PYOPENJTALK_BUILD_VERSION=$VERSION python setup.py develop sdist
+python -m build --sdist
```

Comment:
Running `setup.py` directly is deprecated.
And there is no need to set environment variables.